### PR TITLE
Resurrected CompactionDriverTest resolving TODO in PreDeleteTable

### PR DESCRIPTION
**Pull request by dlmarion**
The TODO asked if the the delete marker in ZooKeeper was still being used / tested. It is still being used by the CompactionDriver. However, the CompactionDriverTest class was deleted at some point which tests it. I resurrected and fixed the test.

### CODE REVIEW

File: server/manager/src/main/java/org/apache/accumulo/manager/tableOps/compact/CompactionDriver.java [hunk 1]
```diff
@@ -129,7 +129,8 @@ public long isReady(FateId fateId, Manager manager) throws Exception {
     return sleepTime;
   }
 
-  private boolean isCancelled(FateId fateId, ServerContext context)
+  // visible for testing
+  protected boolean isCancelled(FateId fateId, ServerContext context)
```
Comment (DomGarguilo) [inline server/manager/src/main/java/org/apache/accumulo/manager/tableOps/compact/CompactionDriver.java:133]:
```suggestion
  // visible for testing
  @VisibleForTesting
  protected boolean isCancelled(FateId fateId, ServerContext context)
```
Could add this annotation here

**Comment by DomGarguilo** [comment]
> I resurrected and fixed the test.

When was this removed?

**Comment by dlmarion** [comment]
> > I resurrected and fixed the test.
> 
> When was this removed?

I'm not sure. I think it exists in `main`, so it must have been removed as part of development in `elasticity` for some reason.

**Comment by dlmarion** [comment]
CompactionDriverTest was removed in ff5148a8c65d6ffa036e82330fa42378fc5dd9af

**Pull request merged**